src/styles/flex.scss

```diff
@@ -0,0 +1,212 @@
+@import './new-settings';
+
+$block: #{$fd-namespace}-flex;
+
+$fd-flex-spacings: (
+  "tiny": 0.5rem,
+  "small": 1rem,
+  "medium": 2rem,
+  "large": 3rem
+);
+
+.#{$block} {
+  // Properties for the Parent (flex container)
+  @mixin fd-flex($direction: row) {
+    display: flex;
+    flex-direction: $direction;
+
+    @content;
+  }
+
+  @include fd-flex();
+
+  // ----------- FLEX ALIGN CONTENT -----------
+  @mixin fd-flex-align-content($type: normal) {
+    display: flex;
+    flex-wrap: wrap;
+    align-content: $type;
+
+    @content;
+  }
+
+  @each $type in (flex-start flex-end center space-between space-around space-evenly stretch start end baseline) {
```

Comment:
I think it would be better if you moved these values to some variable above and then used it here, much easier read